a3p-integration/proposals/b:replace-electorate/replaceElectorate.test.js

```diff
@@ -0,0 +1,77 @@
+import test from 'ava';
+import 'ses';
+import '@endo/init';
+import {
+  agops,
+  ATOM_DENOM,
+  getISTBalance,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+  openVault,
+  queryVstorage,
+  USER1ADDR,
+} from '@agoric/synthetic-chain';
+import {
+  acceptInvitation,
+  marshaller,
+  proposeVaultDirectorParamChange,
+  voteOnProposedChanges,
+  queryVstorageFormatted,
+} from './agoric-tools.js';
+import { waitUntil } from './utils.js';
+
+test('new committee should be able to vote', async t => {
```

Comment:
I suggest splitting this into 2 tests:

 - a test for accepting invitations, which is specific to this upgrade, and belongs here in `b:replace...`
 - a test for making vault parameter changes, which should be tested on every upgrade, so belongs in `z:acceptance`